### PR TITLE
[JENKINS-54375] Use JFR tag instead of commit

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-54375](https://issues.jenkins-ci.org/browse/JENKINS-54375)

Once JFR has been released, we can use the tag on the demo.

@oleg-nenashev @raul-arabaolaza @fcojfernandez

### CODE REVIEW

File: custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/Builder.java [hunk 1]
```diff
@@ -259,7 +259,6 @@ private void buildIfNeeded(@Nonnull DependencyInfo dep, @Nonnull String packagin
                                @CheckForNull List<String> extraMavenArgs)
             throws IOException, InterruptedException {
 
-        String properties;
```
Comment (oleg-nenashev) [inline custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/Builder.java]: unrelated ?

Comment (varyvol) [reply custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/Builder.java]: Yes, unused variable.
